### PR TITLE
set comment status and filter by status

### LINKED ISSUE #2851: allow sorting discussions by status

### PR DESCRIPTION
**Pull request by Entkenntnis**

### CODE REVIEW

File: src/components/comments/comment-area-all-threads-thread.tsx [hunk 1]
```diff
@@ -70,32 +62,37 @@ export function CommentAreaAllThreadsThread({
...
+                submitEvent('thread_status_set')
+                onMutate()
+                void mutateStatus({
+                  id: [thread.id],
+                  status: e.target.value as CommentStatus,
+                })
+              }}
+            >
+              <option value="open" className="bg-white">
+                offen
```
Comment (hejtful) [inline src/components/comments/comment-area-all-threads-thread.tsx:88]: I guess no translations because the whole feature is still a work in progress?

**Review by hejtful — APPROVED** [review]
Code looks good to me!

I'd be happy to test it as well, if you want. I'd just need a link to an entity with comments.

File: src/components/comments/comment-area-all-threads-thread.tsx (hunk 1 above)
Comment (Entkenntnis) [reply src/components/comments/comment-area-all-threads-thread.tsx:88]: we only use this feature for the german language instance, so all of the UI is not translated

**Comment by Entkenntnis** [comment]
You can test it locally under http://localhost:3000/discussions

As soon as the API is deployed, we can continue the testing in staging.

**Pull request merged**